SWIG/blackformula.i

```diff
@@ -0,0 +1,115 @@
+
+/*
+ Copyright (C) 2017 Wojciech Ślusarski
+
+ This file is part of QuantLib, a free-software/open-source library
+ for financial quantitative analysts and developers - http://quantlib.org/
+
+ QuantLib is free software: you can redistribute it and/or modify it
+ under the terms of the QuantLib license.  You should have received a
+ copy of the license along with this program; if not, please email
+ <quantlib-dev@lists.sf.net>. The license is also available online at
+ <http://quantlib.org/license.shtml>.
+
+ This program is distributed in the hope that it will be useful, but WITHOUT
+ ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ FOR A PARTICULAR PURPOSE.  See the license for more details.
+*/
+
+#ifndef quantlib_black_formula_i
+#define quantlib_black_formula_i
+
+
+%{
+using QuantLib::blackFormula;
+using QuantLib::blackFormulaImpliedStdDev;
+using QuantLib::blackFormulaCashItmProbability;
+using QuantLib::bachelierBlackFormula;
+using QuantLib::bachelierBlackFormulaImpliedVol;
+%}
+
+
+%rename(blackFormula) blackFormulaPtr;
```

Comment:
This mechanism is to hide pointers, and you don't need it for these functions.
You can remove the `%rename`s and just copy the declaration of the functions from the C++ header to the SWIG interface.

Reply:
Done
The fault was due to improper declaration with %inline. It was resulting with errors related to ambiguity related to other declaratoins like in header files related to vannavolga interpolation and barrier engines.
Now it compiles and works fine.